addons/realisticnames/stringtable.xml

```diff
@@ -3216,5 +3232,85 @@
             <Chinese>"馬卡洛夫"手槍</Chinese>
             <Chinesesimp>"马卡洛夫"手枪</Chinesesimp>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_lsv_01_armed">
+            <English>Polaris DAGOR (XM312)</English>
```

Comment:
Bei neuen Übersetzungen und solange es nicht eindeutig in allen Sprachen gleich ist, einfach nur die deutsche bzw. auch englische Übersetzung (und sonst alle Sprachen derer du mächtig bist 😁 ) hinzufügen und die anderen weglassen.

Reply:
Ich habe das geändert: a4a9f63dfbc012f0929fefb6eb564c8e90edeef9